api/apps/articles/test/model/index/persistence.test.coffee

```diff
@@ -417,6 +417,17 @@ describe 'Article Persistence', ->
         article.super_article.footer_title.should.equal 'Footer Title'
         done()
 
+    it 'saves super sub articles', (done) ->
+      Article.save {
+        author_id: '5086df098523e60002000018'
+        is_super_article: false
+        is_super_sub_article: true
```

Comment:
Since this is "derived data" or a "computed property", I think it'd be better if we generated this field on the fly. e.g. in the [`present` function](https://github.com/artsy/positron/blob/master/api/apps/articles/model/index.coffee#L142) you can use... 

```coffeescript
superArticleCount = await db.articles.find(_id: article._id).count()
# ...
_.extend article,
  # ...
  is_super_sub_article: superArticleCount > 0
```